### ISSUE TITLE
Duplicate: Charmed Truly

### ISSUE THREAD

**Issue by XDelta** [AdmiralGT/botc-scripts#488]
Charmed Truly - v2.0.0
https://botcscripts.com/script/565

Duplicate of https://botcscripts.com/script/325 which also has more of the older versions of the script.

**Comment by AdmiralGT** [comment]
Hmm, I agree that the Charmed Truly - v2.0.0 is a duplicate but they are both owned by individuals (different ones) but the duplicate does have favourites so I don't simply want to delete it.

I'll need to do some work on this to transfer favourites from the duplicate to the original. Sadly I won't be able to do that until the new year so they will both remain around for some time.

**Comment by XDelta** [comment]
Yeah no worries, figured I should point out dupes to be sorted out eventually
